### ISSUE TITLE
new dataset: HPIDS 2015 TPWD

### ISSUE THREAD

**Issue by mitchellryant** [TNRIS/tnris.org#794]
WPD for incoming new dataset TOP 2015 TPWD areas
- [x] Add data to measures spreadsheet @gmullins
- [x] Route Data Shipment @gmullins
- [x] Data Processing [renaming file, data organization, coverage map] @mitchellryant
- [x] Quality Assurance @gmullins
- [x] Download Prep [zip, copy to s3 bucket] @mitchellryant
- [x] Move data to production shares @mitchellryant
- [x] Add to Data Download @wilsaj https://github.com/TNRIS/data-download/issues/63
- [x] Data Publication [Update data catalog sheet, create data catalog images, supplemental reports staging] @gmullins
- [x] Data Story @gmullins
- [x] Final Publication @christoball @gmullins

**Comment by gmullins** [comment]
Note to self:
Location for creating data catalog images: M:\Admin\Website\DataCatalogImages
datacatalogimage.mxd

**Comment by gmullins** [comment]
QA on hold.
Waiting on Metadata for Hueco and Powderhorn.
Waiting on Accuracy reports for all three areas.

**Comment by gmullins** [comment]
All metadata is in.

**Comment by christoball** [comment]
closed in #913
